.github/WORKFLOWS/pr-labels-check.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2018 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: PR Labels Check
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+jobs:
+  test_approved_or_docs:
+    name: Test approved or docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: danielchabr/pr-labels-checker@master
+        id: checkLabel
+        with:
+          hasSome: PR:tested,PR:no testing needed,docs
```

Comment:
2 questions. Why is there `PR` in front of the name and is it really `no testing needed`?
<img width="313" alt="Screenshot 2021-03-08 at 16 53 46" src="https://user-images.githubusercontent.com/17467768/110345536-57d3ff80-8037-11eb-98eb-0fd33df22a73.png">

Reply:
PR - to signal, these labels are PR related.
Yes the other label is `PR:no testing needed`. I suggest for cases where testing is really not needed. The current update of license being a perfect example.

If you thing the prefixed could be better or should be none, please say so. For the record, because of not having the labels finalized I haven't created/modified them yet. (I wanted to add this kind of remark to the PR description. Just forgot as usually 😊 )

Reply:
I think I get it. You plan to create new or edit existing labels, that is why it is not aligned.